scheduler/src/cook/config.clj

```diff
@@ -403,6 +419,7 @@
                          (throw (ex-info "You enabled nrepl but didn't configure a port. Please configure a port in your config file." {})))
                        ((util/lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))
      :pools (fnk [[:config {pools nil}]]
+              (guard-invalid-gpu-config pools)
```

Comment:
You can destructure valid-gpu-models here using fnk and just pass that to guard-invalid-gpu-config, then it doesn't need to destructure.

Reply:
talked on slack, passed `(:valid-gpu-models pools)` instead of `pools`